src/algebra/periodic.lean

```diff
@@ -47,6 +48,28 @@ lemma periodic.div [has_add α] [has_div β] (hf : periodic f c) (hg : periodic
   periodic (f / g) c :=
 by simp * at *
 
+lemma periodic.const_smul [add_comm_monoid α] [division_ring γ] [module γ α]
```

Comment:
I believe lemmas for `group` are usually unprimed, and corresponding lemmas for `group_with_zero` (or ring/field) are primed.